src/cumulativeSum.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import {Tensor, sizeOfShape} from './lib/tensor.js';
+import {validateCumulativeSumParams} from './lib/validate-input.js';
+
+/**
+ * Computes the cumulative sum of the input tensor along the specified axis.
+ * @param {Tensor} input
+ * @param {number} axis
+ * @param {MLCumulativeSumOptions} options
+ * @return {Tensor}
+ */
+export function cumulativeSum(input, axis, {exclusive = 0, reverse = 0} = {}) {
```

Comment:
```suggestion
export function cumulativeSum(input, axis, {exclusive = false, reverse = false} = {}) {
```
Refer to https://chromium-review.googlesource.com/c/chromium/src/+/5845069/9/third_party/blink/renderer/modules/ml/webnn/ml_graph_builder.idl#66
```spec
dictionary MLCumulativeSumOptions : MLOperatorOptions {
    boolean exclusive = false;
    boolean reversed = false;
};

```

Reply:
done